### PR TITLE
feat: Start Erigon and Lighthouse

### PR DESCRIPTION
**Pull request by Wolmin**
This PR features support for starting erigon+lighthouse clients. IMPORTANT: please keep in mind that there is no support for lighthouse validator yet!

### CODE REVIEW

**Comment by Hugoo** [comment]
CU-863gevaza

File: cmd/lukso/dependencies.go [hunk 1]
```diff
@@ -73,93 +88,138 @@ var (
...
 		},
-		prysmDevnetChainConfigDependencyName: {
+		devnetChainConfigDependencyName: {
 			baseUrl:  "https://raw.githubusercontent.com/lukso-network/network-configs/main/devnets/3030/shared/config.yaml",
-			name:     prysmDevnetChainConfigDependencyName,
+			name:     devnetChainConfigDependencyName,
 			filePath: devnetConfig + "/" + chainConfigYamlPath,
 		},
+		gethDevnetConfigName: {
+			baseUrl:  "https://raw.githubusercontent.com/lukso-network/network-configs/main/devnets/3030/geth/geth.toml",
```
Comment (Hugoo) [inline cmd/lukso/dependencies.go:199]: do we still use devnet 3030?

**Review by Hugoo — COMMENTED** [review]
We also need to update the README to explain that it now support these 2 new clients

image

**Comment by fhildeb** [comment]
Documentation for README added here: #130

**Review by fhildeb — APPROVED** [review]

**Pull request merged**